index.js

```diff
@@ -5,5 +5,5 @@
 module.exports = function(content) {
 	this.cacheable && this.cacheable();
 	this.value = content;
-	return "module.exports = " + JSON.stringify(content);
+	return "export default " + JSON.stringify(content);
```

Comment:
I had to do

```js
import * as foo from './foo'
```
which is very wrong... If this will correct behavior, please accept...

Confirmed working with webpack v2.2.1.